README.md

```diff
@@ -158,7 +163,40 @@ yarn start
 open localhost:3000
 ```
 
+## Docker Quickstart
+To start Unity Catalog in Docker Compose in one command, install the latest version of [Docker Desktop](https://www.docker.com/products/docker-desktop/) and run the following:
+
+```sh
+docker compose up -d
+```
+
+This starts the Unity Catalog UI and backend. You can access the UI at http://localhost:3000 and the backend at http://localhost:8080.
```

Comment:
This "backend" caught my attention. Is this "the Unity Catalog server" perhaps? What else?

You may want to write "all the Unity Catalog components" and link it to the respective architecture page where they're described (if there's any = no idea).

Reply:
To clarify, does `./bin/start-uc-server` start all the components? Because that is the CMD for the image.

Reply:
All the default components of the UC Localhost Server, yes. I think this backend term comes more from Docker Compose where it's used. Correct?